rdmo/management/utils.py

```diff
@@ -0,0 +1,16 @@
+from rdmo.domain.models import Attribute
+
+
+def replace_uri_in_template_string(template: str, source: Attribute, target: Attribute) -> str:
+    new_lines = []
+    path_changed = source.path != target.path
+    for line in template.splitlines():
+        new_line = line
+        if source.uri in line:
+            new_line = new_line.replace(source.uri, target.uri)
```

Comment:
This should replace the uri with `'` or `"` included (maybe with `re.sub`?), I just merged `.../test` with `.../test2`, and this will also replace `.../test2` with `.../test22` and so on.